libs/sdk-ui-dashboard/src/presentation/dashboard/components/DashboardScreenSizeContext.tsx

```diff
@@ -25,6 +37,20 @@ const getScreenSize = (ref: React.RefObject<HTMLDivElement>): ScreenSize => {
     return "xs";
 };
 
+function setNewSize(
+    screenSize: string | undefined,
+    setScreenSize: React.Dispatch<React.SetStateAction<ScreenSize | undefined>>,
+    dispatch: React.Dispatch<AnyAction>,
+) {
+    const newSize = getScreenSize();
```

Comment:
possibly change this to `const currentScreenSize = detectCurrentScreenSize();` or `getCurrentScreenSize`

Reply:
fixed